README.md

```diff
@@ -114,6 +114,7 @@ The following input parameters are also supported for the Endor Labs GitHub acti
 | Flags | Description |
 | :-- | :-- |
 | `additional_args` | Use additional_args to add custom arguments to the endorctl scan command. |
+| `disable_phantom` | Set to `false` to enable phantom dependency analysis. (Default: `true`) |
```

Comment:
The double negative is confusing. Can we please change this to `phantom_dependencies` with the default being `false`

Reply:
@ewokcami updated the flag name in the monorepo to be more clear. Once that is approved and merged, will update it here as well.